dev-libs/lief/lief-0.11.5.ebuild

```diff
@@ -0,0 +1,125 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{8..10} pypy3 )
+
+# Upstream has two alternate approaches to building its Python API:
+# 1. A working "CMakeList.txt" only supporting a single Python target.
+# 2. A non-working "setup.py" supporting multiple Python targets but internally
+#    invoking CMake in mostly non-configurable (and thus broken) ways.
+# We choose the working approach.
+inherit python-single-r1 cmake
+
+DESCRIPTION="Library to instrument executable formats"
+HOMEPAGE="https://lief.quarkslab.com"
+SRC_URI="https://github.com/lief-project/LIEF/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="c examples +python static-libs"
+REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
+
+# See "cmake/LIEFDependencies.cmake" for C and C++ dependencies.
+BDEPEND="
+	python? (
+		$(python_gen_cond_dep '
+			>=dev-python/setuptools-31.0.0[${PYTHON_USEDEP}]
+		')
+	)
+"
+#FIXME: Add after bumping to the next stable release:
+#	>=dev-libs/spdlog-1.8.5
+RDEPEND="python? ( ${PYTHON_DEPS} )"
+DEPEND="${RDEPEND}"
+
+# LIEF tests are non-trivial (if not infeasible) to run in the general case.
+# For example, "tests/CMakeLists.txt" implies all USE flags must be enabled:
+#     if (NOT LIEF_ELF OR NOT LIEF_PE OR NOT LIEF_MACHO)
+#       message(FATAL_ERROR "Tests require all LIEF's modules activated" )
+#     endif()
+RESTRICT="test"
+
+S="${WORKDIR}/LIEF-${PV}"
+
+DISTUTILS_OPTIONAL=1
```

Comment:
Since we are not inheriting `distutils-r1` this is not doing anything, right?

Reply:
`</sigh>`

That vestigial holdover from the bad old days (...so, last week) in which I still believed in LIEF's `setup.py` is, indeed, vestigial. Andrew's All-Seeing Eye clarifies all. I'll cauterize the rot away.